in_toto/runlib.go

```diff
@@ -339,22 +339,37 @@ func InTotoRun(name string, runDir string, materialPaths []string, productPaths
 		return nil, err
 	}
 
-	link := Link{
-		Type:        "link",
-		Name:        name,
-		Materials:   materials,
-		Products:    products,
-		ByProducts:  byProducts,
-		Command:     cmdArgs,
-		Environment: map[string]interface{}{},
+	// FIXME: replace when Attestation Framework support is fully implemented
+	var link Link
+	if attest {
+		// FIXME
+
+		/* Pseudocode:
+
+		   provenance = provV1.GenerateValidPredicate()
+		   provStruct = common.PredicatePbToStruct(provenance)
+
+		   subject = v1.RDListFromRecord(products)
+		   statement = v1.GenerateValidStatement(subject, "https://slsa.dev/provenance/v1", provStruct)
+		*/
+		return nil, errors.New("in-toto attestation support not implemented yet")
+	} else {
+		link = Link{
+			Type:        "link",
+			Name:        name,
+			Materials:   materials,
+			Products:    products,
+			ByProducts:  byProducts,
+			Command:     cmdArgs,
+			Environment: map[string]interface{}{},
+		}
 	}
 
 	if useDSSE {
```

Comment:
FWIW, this should apply if `useDSSE` or `attest` is set. We don't want `attest` to go to the legacy sig wrapper.

Reply:
Ok! I wasn't sure if this was the intent here. I'll change this.

Reply:
```suggestion
	if useDSSE || attest {
```